x-pack/legacy/plugins/apm/server/routes/settings.ts

```diff
@@ -20,13 +20,19 @@ import { transactionSampleRateRt } from '../../common/runtime_types/transaction_
 // get list of configurations
 export const agentConfigurationRoute = createRoute(core => ({
   path: '/api/apm/settings/agent-configuration',
-  handler: async req => {
-    await createApmAgentConfigurationIndex(core.http.server);
+  handler: async (req, _, h) => {
```

Comment:
are we using `h`? can't see it being used below.

Reply:
my mistake for leaving this in